desk/app/channels-server.hoon

```diff
@@ -1134,116 +1113,118 @@
   =;  [result=(each event:h tang) effects=(list effect:h)]
     [result (run-hook-effects effects nest)]
-  =/  current-event  event
   =|  effects=(list effect:h)
   =/  order  (~(got by order.hooks) nest)
   =/  channel  `[nest (~(got by v-channels) nest)]
-  =/  =context:h  (get-hook-context channel *config:h)
+  =/  =bowl:h  (get-hook-bowl channel *config:h)
   |-
   ?~  order
-    [&+current-event effects]
+    [&+event effects]
   =*  next  $(order t.order)
   =/  hook  (~(got by hooks.hooks) i.order)
-  =/  ctx  context(hook hook, config (~(gut by config.hook) nest ~))
+  =.  bowl  bowl(hook hook, config (~(gut by config.hook) nest ~))
   =/  return=(unit return:h)
-    (run-hook:utils [current-event ctx] hook)
+    (run-hook:utils [event bowl] hook)
   ?~  return  next
   =*  result  result.u.return
   =.  effects  (weld effects effects.u.return)
   =.  hooks.hooks  (~(put by hooks.hooks) i.order hook(state new-state.u.return))
   ?:  ?=(%denied -.result)
     [|+~[(fall msg.result default)] effects]
-  =.  current-event  new.result
+  =.  event  event.result
   next
-++  wakeup-hook
+++  wake-hook
   |=  =(pole knot)
   ^+  cor
-  ?+  pole  ~|(bad-arvo-take/pole !!)
-      [%delayed id=@ ~]
-    =/  =id:h  (slav %uv id.pole)
-    ?~  delay=(~(get by delayed.hooks) id)  cor
+  ?+  pole  ~|(bad-arvo-take+pole !!)
+      [%waiting id=@ ~]
+    =/  id=id-hook:h  (slav %uv id.pole)
+    ?~  wh=(~(get by waiting.hooks) id)  cor
     ::  make sure we clean up
-    =.  delayed.hooks  (~(del by delayed.hooks) id)
+    =.  waiting.hooks  (~(del by waiting.hooks) id)
     ::  ignore premature fires
-    ?:  (lth now.bowl fires-at.u.delay)  cor
-    =*  origin  origin.u.delay
-    =/  hook  (~(got by hooks.hooks) hook.u.delay)
+    ?:  (lth now.bowl fires-at.u.wh)  cor
+    =*  origin  origin.u.wh
+    =/  hook  (~(got by hooks.hooks) hook.u.wh)
     =/  config  ?@(origin ~ (~(gut by config.hook) origin ~))
-    =/  args  [[%wake +.u.delay] "delayed hook" origin config]
-    ho-abet:(ho-run-single:(ho-abed:ho-core hook.u.delay) args)
+    =/  args  [[%wake +.u.wh] "waiting hook" origin config]
+    ho-abet:(ho-run-single:(ho-abed:ho-core hook.u.wh) args)
   ::
       [%cron id=@ kind=?(%chat %diary %heap) ship=@ name=@ ~]
-    =/  =id:h  (slav %uv id.pole)
+    =/  id=id-hook:h  (slav %uv id.pole)
     =/  =origin:h  [kind.pole (slav %p ship.pole) name.pole]
     ::  if unscheduled, ignore
-    ?~  crons=(~(get by crons.hooks) id)  cor
-    ?~  cron=(~(get by u.crons) origin)  cor
+    ?~  cron=(~(get by crons.hooks) id)  cor
+    ?~  job=(~(get by u.cron) origin)  cor
     ::  ignore premature fires
-    ?:  (lth now.bowl next.schedule.u.cron)  cor
-    =.  next.schedule.u.cron
+    ?:  (lth now.bowl next.schedule.u.job)  cor
+    =.  next.schedule.u.job
+      ::
       ::  we don't want to run the cron for every iteration it would
       ::  have run 'offline', so we check here to make sure that the
       ::  next fire time is in the future
```

Comment:
```suggestion
      ::  we don't want to run the cron for every iteration it would
      ::  have run 'offline', so we check here to make sure that the
      ::  next fire time is in the future
      ::
```